Library/Tests/Koala/KoalaTests.swift

```diff
@@ -29,8 +29,8 @@ final class KoalaTests: XCTestCase {
     XCTAssertNotNil(properties?["model"])
     XCTAssertEqual(device.systemName, properties?["os"] as? String)
     XCTAssertEqual(device.systemVersion, properties?["os_version"] as? String)
-    XCTAssertEqual(screen.bounds.width, properties?["screen_width"] as? CGFloat)
-    XCTAssertEqual(screen.bounds.height, properties?["screen_height"] as? CGFloat)
+    XCTAssertEqual(UInt(screen.bounds.width), properties?["screen_width"] as? UInt)
+    XCTAssertEqual(UInt(screen.bounds.height), properties?["screen_height"] as? UInt)
```

Comment:
This was a little weird. Not sure why I need to do the `UInt` casting. Anyone have ideas?